tools/ci_build/github/azure-pipelines/orttraining-py-packaging-pipeline-cuda115.yml

```diff
@@ -1,5 +1,11 @@
 trigger: none
 
+parameters:
```

Comment:
Instead of having a parameter, you can create a new build job. I think we want to have to debug packages every day and publish them ?

Reply:
@askhade Do we need to build and publish daily?

Reply:
@snnn I think we only need an on demand package. So, we don't need to publish this every day. I think it is ok to have a parameter that can trigger the build, and the users can pull the whl file after the build is complete.

Reply:
On demand is good for debug packages